ESSArch_PP/static/lang/sv.json

```
@@ -45,11 +45,10 @@
     "OK": "Ok",
     "CANCEL": "Avbryt",
     "SUBJECT": "Ämne",
-    "PARAMS": "Params",
+    "PARAMETERS": "Parametrar",
     "STEP": "Step",
```

STEP should be translated in Swedish:

```suggestion
     "STEP": "Steg",
```